lib/services/ui/chat/conversation_view_controller.dart

```diff
@@ -52,6 +52,7 @@ class ConversationViewController extends StatefulController with GetSingleTicker
   final RxBool recipientNotifsSilenced = false.obs;
   bool showingOverlays = false;
   bool _subjectWasLastFocused = false; // If this is false, then message field was last focused (default)
+  StreamSubscription<dynamic>? cancelTypingIndicator;
```

Comment:
Signal to finish typing. Can be triggered by backends to signal the other person is no longer typing

Reply:
where is this used? typically you should just have a listener on the RxBool showTypingIndicator which will let you know when the variable is updated